src/app/functions/server/onerep.ts

```diff
@@ -20,6 +20,13 @@ import {
 } from "../../../db/tables/featureFlags";
 import { logger } from "./logging";
 
+export const monthlyScanQuota = parseInt(
+  (process.env.MONTHLY_SCANS_QUOTA as string) || "0",
+);
+export const monthlySubscriberQuota = parseInt(
+  (process.env.MONTHLY_SUBSCRIBERS_QUOTA as string) || "0",
+);
```

Comment:
Should these variable names be plural'ed to match their respective env vars?

| varible | env | &nbsp;
|----|----|----
`monthlyScanQuota` | `MONTHLY_SCANS_QUOTA` | (scan vs scans)
`monthlySubscriberQuota` | `MONTHLY_SUBSCRIBERS_QUOTA` | (subscriber vs subscribers)

Reply:
thanks fixed